Makefile

```diff
@@ -278,3 +278,22 @@ KUTTL=$(GOBIN)/kuttl
 else
 KUTTL=$(shell which kubectl-kuttl)
 endif
+
+golangci:
+ifeq (, $(shell which golangci-lint))
+	@{ \
+	set -e ;\
+	KUTTL_TMP_DIR=$$(mktemp -d) ;\
```

Comment:
I think we should not refer to kuttl here. Plus, do we actually need to do go mod init? - go install should work without that just fine.